src/server/routes/__tests__/catchments.unit.spec.js

```diff
@@ -0,0 +1,70 @@
+import { Catchment } from '../../../entities/index.js'
+import logger from '../../../utils/logger-utils.js'
+import routes from '../catchments.js'
+
+jest.mock('../../../entities/index.js')
+jest.mock('../../../utils/logger-utils.js')
+
+describe('catchments.unit', () => {
+  describe('GET /catchments', () => {
+    const h = {
+      response: jest.fn().mockReturnThis(),
+      code: jest.fn()
+    }
+
+    afterEach(() => {
+      jest.clearAllMocks()
+    })
+
+    it('should return all catchments with a 200 status code', async () => {
+      const catchmentData = [
+        {
+          id: '2',
+          name: 'Adur',
+          createdAt: '2018-11-07T10:00:00.000Z',
+          updatedAt: '2018-11-07T10:00:00.000Z'
+        },
+        {
+          id: '6',
+          name: 'Annas',
+          createdAt: '2018-11-07T10:00:00.000Z',
+          updatedAt: '2018-11-07T10:00:00.000Z'
+        }
+      ]
+      Catchment.findAll.mockResolvedValueOnce(catchmentData)
+
+      const handler = routes[0].options.handler
+      await handler({}, h)
+
+      expect(Catchment.findAll).toHaveBeenCalled()
```

Comment:
This looks like it's enforcing a particular implementation - it's fine to mock out Catchment.findAll, but I don't think a test should fail if it's not called. We're putting in sausage meat and sausage skins at one end, then expecting sausages at the other end. How this is accomplished is not really the business of the test, only that it is.